ocaml/xapi/xapi_globs.ml

```diff
@@ -1017,6 +1017,14 @@ let conn_limit_unix = ref 1024
 
 let conn_limit_clientcert = ref 800
 
+let stunnel_cache_max_age = ref (180. *. 60.)
+
+let stunnel_cache_max_idle = ref (5. *. 60.)
```

Comment:
These aren't referenced anywhere. Only the ones in Stunnel_cache are used.

Reply:
Yes, that is to set this ref from the config file. But nothing actually uses it.

Reply:
Yes, same bug here too https://github.com/xapi-project/xen-api/pull/6119/files

Reply:
I removed the extra values in xapi_globs.ml, and I'm setting the ones in stunnel directly now.
(I don't know what that'll mean for the yangtze backport where these are in separate repos likely), but we'll find out when we do the backport.